### PR TITLE
#1 add the basic of protocol

### PR DESCRIPTION
**Pull request by Enrique199413**

### CODE REVIEW

File: delivery.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Delivery Protocol
+
+For basic proyects `Level Zero` in blick scale consider deliver:
+
+- `Amazon Account`
+- `Github Repositorie`
```
Comment (jresendiz27) [inline delivery.md]: it's repository.

File: delivery.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+- `Related third accounts`
```
Comment (jresendiz27) [inline delivery.md]: did you mean third party accounts?

Comment (Enrique199413) [inline delivery.md]: For external accounts for example `mailchimp`

File: delivery.md [hunk 3]
```diff
@@ -0,0 +1,7 @@
+# Delivery Protocol
+
+For basic proyects `Level Zero` in blick scale consider deliver:
```
Comment (AlanSanchezP) [inline delivery.md]: Change to "to deliver"

File: delivery.md [hunk 4]
```diff
@@ -0,0 +1,7 @@
+# Deliver Protocol
```
Comment (jresendiz27) [inline delivery.md]: it's delivery, deliver it's a verb.

File: delivery.md (hunk 3 above)
Comment (Enrique199413) [inline delivery.md]: Change for Delivery because is the correct

**Pull request merged**